packages/flutter/lib/src/material/navigation_drawer.dart

```diff
@@ -193,6 +193,7 @@ class NavigationDrawerDestination extends StatelessWidget {
     required this.icon,
     this.selectedIcon,
     required this.label,
+    this.disabled = false,
```

Comment:
You should be using a positive argument here. So `enabled` instead of `disabled`.

See https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-double-negatives-in-apis